server/lib/notifications.js

```diff
@@ -201,14 +201,48 @@ async function notifyByEmail(activity) {
 
     case activityType.COLLECTIVE_UPDATE_PUBLISHED:
       twitter.tweetActivity(activity);
-      activity.data.update = await models.Update.findByPk(activity.data.update.id, {
```

Comment:
Why getting rid of this line, was it not needed?

Reply:
we only need `activity.data.update.collectiveId`, which we already have access to.

Reply:
ah yes , `info.html` content from here was used in email templates. That's why it went missing.